### ISSUE TITLE
[1 point] Splitted static & parametrized routers

### ISSUE THREAD

**Issue by toby3d** [gramework/gramework#30]
Gramework can't set route with path argument on root: 
```bash
...
   • registering GET /        
   • registering GET /editor  
   • registering POST /publish
   • registering GET /:id     
panic: wildcard route ':id' conflicts with existing children in path '/:id'
```
So, I can't use dynamic paths like `/:slug` or `/:id-:slug` in my project for serve blog-posts (eg. `/1234-hello-world`, `/my-custom-page`).

**Comment by kirillDanshin** [comment]
relates to #14

**Comment by yakud** [comment]
@kirillDanshin it's same with https://github.com/gramework/gramework/pull/57 ?

**Comment by kirillDanshin** [comment]
yeah, finally closed via #57
